### DISCUSSION TITLE
How to play rtp_mpegts stream on client side

### DISCUSSION THREAD

**Discussion by donmezburak** [pion/webrtc#2650] [Q&A] [answered]
Hello, I have ffmpeg stream with -f rtp_mpegts option. Pion go part recieves stream however video cannot work.

Is there any way to play rtp_mpegts stream on client side video tag?

Thanks

**Comment by Sean-Der** [comment]
Hey @donmezburak 

Browser's don't support mpegts via RTP. What codecs are you forwarding? 

You will need to unpack the mpegts and then re-packetize unfortunately. Happy to help with the code for that! https://github.com/flavioribeiro/donut/ does most of the work already!

Instead of mpegts via SRT it will be via RTP.

**Comment by donmezburak** [comment]
Hi Sean,

I am using **libx264** codec. Thanks for your answer. actually we are serving live stream of computer screens and I want to record stream on server side but I could not due to rtp protocol. Probably I changed our protocol option.
Thanks

**Comment by Sean-Der** [comment, marked as answer]
@donmezburak Since you are using ffmpeg and RTP could you do something like [rtp-forwarder](https://github.com/pion/webrtc/tree/master/examples/rtp-forwarder) instead?

Do you have to use mpegts inside RTP?

**Comment by donmezburak** [comment]
No, I do not have to use mpegts. I just used that to save stream easily. 

You are great man. I will look rtp-forwarder, probably it fixes my issue.
